exporter/debugexporter/config.go

```diff
@@ -32,6 +33,8 @@ type Config struct {
 	// UseInternalLogger defines whether the exporter sends the output to the collector's internal logger.
 	UseInternalLogger bool `mapstructure:"use_internal_logger"`
 
+	QueueConfig exporterhelper.QueueBatchConfig `mapstructure:"sending_queue"`
```

Comment:
Shouldn't this new config option be documented?

Reply:
Realized that a few exporters were missing this config: https://github.com/open-telemetry/opentelemetry-collector/pull/13805